pkg/kubelet/cm/cpumanager/topology_hints_test.go

```diff
@@ -31,21 +31,17 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/cm/topologymanager/bitmask"
 )
 
-func TestGetTopologyHints(t *testing.T) {
-	testPod1 := makePod("fakePod", "fakeContainer", "2", "2")
-	testContainer1 := &testPod1.Spec.Containers[0]
-	testPod2 := makePod("fakePod", "fakeContainer", "5", "5")
-	testContainer2 := &testPod2.Spec.Containers[0]
-	testPod3 := makePod("fakePod", "fakeContainer", "7", "7")
-	testContainer3 := &testPod3.Spec.Containers[0]
-	testPod4 := makePod("fakePod", "fakeContainer", "11", "11")
-	testContainer4 := &testPod4.Spec.Containers[0]
-
-	firstSocketMask, _ := bitmask.NewBitMask(0)
-	secondSocketMask, _ := bitmask.NewBitMask(1)
-	crossSocketMask, _ := bitmask.NewBitMask(0, 1)
+type testCase struct {
+	name          string
+	pod           v1.Pod
+	container     v1.Container
+	assignments   state.ContainerCPUAssignments
+	defaultCPUSet cpuset.CPUSet
+	expectedHints []topologymanager.TopologyHint
+}
```

Comment:
What is the purpose of this refactoring?
Is this type used elsewhere other than this specific test?
Is `returnMachineInfo()` used elsewhere?

If not, I think it is better to leave these definitions scoped inside of the test itself, rather than create a new type /function that clutters the namespace with something that is not reusable.

Reply:
Yes, `returnMachineInfo()` is used for both `GetTopologyHints()` and `GetPodTopologyHints()`. Because of that it is exported in separate function. I will leave it for now as it is.